File: java/sourceafis/src/sourceafis/matching/minutia/EdgeLocation.java
package sourceafis.matching.minutia;
/*
 * Added newly
 */
public class EdgeLocation {

	public short reference;
	public short neighbor;

	public EdgeLocation(int reference, int neighbor) {
		this.reference = (short) reference;
		this.neighbor = (short) neighbor;
	}
}

File: SourceAFIS/Matching/EdgeLocation.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace SourceAFIS.Matching
{
    public struct EdgeLocation
    {
        public int Reference;
        public int Neighbor;

        public EdgeLocation(int reference, int neighbor)
        {
            Reference = (int)reference;
            Neighbor = (int)neighbor;
        }
    }
}
